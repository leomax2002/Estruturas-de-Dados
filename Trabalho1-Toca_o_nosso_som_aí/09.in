add Staralfur
add Untitled#8
add Olsen Olsen
list
current
play
ended
current
stop
current
list
fight