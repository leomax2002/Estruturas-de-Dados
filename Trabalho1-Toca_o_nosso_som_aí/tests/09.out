Staralfur,Untitled#8,Olsen Olsen
Staralfur
Untitled#8
Untitled#8
Staralfur,Untitled#8,Olsen Olsen
Jedi Wagner, assuma o comando!